src/main/webapp/pages/projects/partials/samples.tmpl.html

```diff
@@ -29,48 +29,49 @@
 			<span th:text="#{project.samples.nav.remove}"></span>
 		</button>
 	</div>
-
+	<div id="linker"></div>
 	<!-- SAMPLE EXPORT -->
-	<div class="btn-group btn-group-sm" uib-dropdown="">
-		<button id="export-btn" class="btn btn-default" uib-dropdown-toggle=""
-		        ng-disabled="samplesCtrl.disabled.lessThanOne">
-			<span th:text="#{project.samples.export.label}">Export</span>
-			<i class="fa fa-caret-down" aria-hidden="true"></i>
-		</button>
-		<ul class="ng-cloak" uib-dropdown-menu="" role="menu" aria-labelledby="export-btn">
-			<li role="menuitem">
-				<a id="exportDownloadBtn" href="#">
-					<i class="fa fa-download fa-fw" aria-hidden="true"></i>
-					<span th:text="#{form.download}">_Download</span>
-				</a>
-			</li>
-			<!--TODO: Fix tooltip-->
-			<li role="menuitem" th:if="${linkerAvailable}" ng-class="{disabled: ! subnav.localSelected}"
-			    ng-attr-uib-tooltip="{{subnav.showuib-tooltip()}}" uib-tooltip-append-to-body="true"
-			    uib-tooltip-placement="right">
-				<a id="exportLinkerBtn" href="#" ng-click="subnav.export.linker()">
-					<i class="fa fa-terminal fa-fw" aria-hidden="true"></i>
-					<span th:text="#{project.samples.export.linker}"></span>
-				</a>
-			</li>
-			<li role="menuitem">
-				<a id="exportNcbiButton" href="#" ng-click="subnav.export.ncbi()">
-					<i class="fa fa-upload fa-fw" aria-hidden="true"></i>
-					<span th:text="#{project.samples.export.ncbi}">_Upload to NCBI_</span>
-				</a>
-			</li>
-			<li role="menuitem" th:if="${galaxyCallback}">
-				<a href="#" ng-click="subnav.export.galaxy(); cart.show=false">
-					<i class="fa fa-cloud-upload fa-fw" aria-hidden="true"></i>
-					<span th:text="#{project.samples.export.galaxy}"></span>
-				</a>
-			</li>
-			<li role="separator" class="divider"></li>
-			<li role="menuitem">
-				<a href="">Excel</a>
-			</li>
-		</ul>
-	</div>
+<!--	<div class="btn-group btn-group-sm" uib-dropdown="">-->
```

Comment:
Any reason this commented out html is here? If not then it can be removed

Reply:
This file should not even exist any more!  Removed in 5513dd21e810924e81fc2dfc6bd7b4e5c5049c50